src/feed/feed.service.integration.test.ts

```diff
@@ -521,6 +521,58 @@ describe('FeedService', () => {
         });
     });
 
+    describe('getGlobalFeedUserId', () => {
+        it('should return the global feed user ID when it exists', async () => {
+            const feedService = new FeedService(client, moderationService);
+
+            const globalAccount = await createInternalAccount(
+                'ap-global-feed.ghost.io',
+            );
+
+            const expectedUser = await client('users')
+                .where('account_id', globalAccount.id)
+                .select('id')
+                .first();
+
+            const userId = await feedService.getGlobalFeedUserId();
+
+            expect(userId).toBe(expectedUser.id);
+        });
+
+        it('should return null when the global feed user does not exist', async () => {
+            const feedService = new FeedService(client, moderationService);
+
+            const userId = await feedService.getGlobalFeedUserId();
+
+            expect(userId).toBe(null);
+        });
+
+        it('should cache the result even if the database record is deleted', async () => {
```

Comment:
nitpick: naming of the test

I think what we really want to test here is that we're caching the result so that _we don't have to a database query on each request_ (and we're using the database record deletion in that test as a mechanism to prove that). 

Here the test description reads to me as "we are not invalidating the cache when the global user is deleted" - which is not something we care about?